CommonMark/CommonMarkAdditionalFeatures.cs

```diff
@@ -21,6 +21,26 @@ public enum CommonMarkAdditionalFeatures
         StrikethroughTilde = 1,
 
         /// <summary>
+        /// The parser will recognize syntax <c>~foo~</c> that will be rendered as <c>&lt;sub&gt;foo&lt;/sub&gt;</c>.
+        /// </summary>
+        SubscriptTilde = 2,
+
+        /// <summary>
+        /// The parser will recognize both <see cref="StrikethroughTilde"/> and <see cref="SubscriptTilde"/>.
+        /// </summary>
+        Tilde = 3,
```

Comment:
I am not sure this option is needed. What if we add another use for tilde - then this option could not mean all of them.